src/parser/parser_utils.cpp

```diff
@@ -25,55 +25,33 @@ namespace parser {
 
 std::string indent(uint num_indent) { return std::string(num_indent, '\t'); }
 
-void inprint(UNUSED_ATTRIBUTE int64_t val, UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%lu", val);
-}
-
-void inprint(UNUSED_ATTRIBUTE float val, UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%f", val);
-}
-
-void inprint(UNUSED_ATTRIBUTE const char* val,
-             UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%s", val);
-}
-
-void inprint(UNUSED_ATTRIBUTE const char* val,
-             UNUSED_ATTRIBUTE const char* val2,
-             UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%s -> %s", val, val2);
-}
-
-void inprintC(UNUSED_ATTRIBUTE char val, UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%c", val);
-}
-
-void inprintU(UNUSED_ATTRIBUTE uint64_t val, UNUSED_ATTRIBUTE uint num_indent) {
-  LOG_TRACE("%lu", val);
-}
-
-void PrintTableRefInfo(TableRef* table, UNUSED_ATTRIBUTE uint num_indent) {
+std::string ParserUtils::GetTableRefInfo(const TableRef* table,
+                                         uint num_indent) {
+  std::string output;
```

Comment:
Use `std::ostringstream` instead of appending to a `std::string`. It's more efficient.